### ISSUE TITLE
File that used to slice no longer slices

### ISSUE THREAD

**Issue by FooltoWorld** [Ultimaker/Cura#19310]
### Cura Version

5.7.2

### Operating System

Windows 11

### Printer

Anycubic kobra Max

### Name abnormal settings

it doesn't slice when using default

### Describe model location

Center of platform, helmet face looks toward front of printer

### Describe your model

Helmet. Cura sliced it before, now it fails to do so

### Add your .zip here ⬇️

[AKMAX_Helldivers 2 basic.zip](https://github.com/user-attachments/files/16045638/AKMAX_Helldivers.2.basic.zip)

**Comment by GregValiant** [comment]
The file didn't upload correctly.  I got the "No model in the file" warning.  Other apps won't open it either.

**Comment by FooltoWorld** [comment]
I can try and re-upload it when I get home from work, or would you prefer
it another way?

**Comment by GregValiant** [comment]
No, just zip it and post it here.  Wait for the upload to complete.  Email does not work when posting files.

**Comment by GregValiant** [comment]
Thanks for the report.
I didn't have any trouble slicing this on my Win 10 laptop.  After a slicing crash, go to the Configuration Folder and zip the Cura.log file and post it here.  This might be a conflict with your installation of Windows.

The Cura team will take a look, but they will need that log file.

**Comment by FooltoWorld** [comment]
Zipping the file is too large to post here, i have uploaded it as a link from my google drive: https://drive.google.com/file/d/1Dp3dBW2ReHtF3kF0OlZP2IQhZ9I_MBTl/view?usp=drive_link
